tests/static/e2e/pages/incidents-page.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Locator, Page } from "@playwright/test"
+import { Routes, orgSlug } from "../routes"
+
+export class IncidentsPage {
+  readonly page: Page
+  readonly route: string
+  readonly Row: Locator
+  readonly FirstRow: Locator
+  readonly OtherRow: Locator
+  readonly NextPage: Locator
+  readonly EditKebab: Locator
+  readonly EditMenu: Locator
+  readonly EditViewEdit: Locator
+  readonly EditCreateReport: Locator
+  readonly EditRunWorkflow: Locator
+  readonly EditDelete: Locator
+
+  constructor(page: Page, incident: string = `dispatch-${orgSlug}-${orgSlug}-2`) {
```

Comment:
The default incident here is a bit fragile to it being present in the table, there's probably a better way.